src/fs/list.js

```diff
@@ -1,5 +1,19 @@
+import {parse} from "path";
+import {readdir} from "node:fs/promises";
+import {filesDirPath} from "./filesDirPath.js";
+
 const list = async () => {
-    // Write your code here 
-};
+  try {
+    const files = await readdir(filesDirPath);
+    for (const file of files) {
+      const {name} = parse(file);
```

Comment:
In the console, the filenames are without extensions. It's not a requirement in the task, but I believe we should show the full filename.